contrib/helm/pganalyze-collector/values.yaml

```diff
@@ -74,10 +83,15 @@ resources:
     memory: 1024Mi
 
 # -- List of volumes to attach to the pod
-volumes: []
+volumes:
+  - name: scratch
+    emptyDir: {}
 
 # -- List of volume mounts to attach to the container
-volumeMounts: []
+volumeMounts:
+  - mountPath: /tmp
+    name: scratch
+    subPath: tmp
```

Comment:
Given the read-only root filesystem setting, I wonder if we also need to create a volume mount for `/state`? (which is whether the state file is written to - whilst that is mainly intended to support restarts outside of a container-based environment, it avoids errors that would likely occur when `/state` is no writeable)

See https://github.com/pganalyze/collector/blob/main/contrib/docker-entrypoint.sh#L23

Reply:
Ah, yes, absolutely! I didn't know about this file